downloads/index.html

```diff
@@ -358,26 +358,23 @@
 
 <p>
   Use <a href="http://git-scm.com/">git</a> to follow the latest changes or
-  <a href="http://code.i3wm.org/i3">view them on this webpage</a>. This command checks out the 'master'
+  <a href="https://www.github.com/i3/i3">view them on this webpage</a>. This command checks out the 'next'
   branch:
 </p>
 
 <pre>
-  $ git clone git://code.i3wm.org/i3
+  $ git clone git@github.com:i3/i3.git
 </pre>
 
 <p>
   The 'master' branch is the current stable version plus bugfixes.<br>
   The 'next' branch is the next version, yet to be released. Use <tt>git
-  checkout next</tt> to switch to the 'next' branch.
+  checkout master</tt> to switch to the 'master' branch.
 </p>
 
 <p>
   If you prefer to download a tarball, or if you cannot use git for whatever reason,
-  you may download the current master branch from
-  <a href="http://code.i3wm.org/i3/snapshot/i3-master.tar.bz2">
-    http://code.i3wm.org/i3/snapshot/i3-master.tar.bz2
-  </a>
+  you can find individual releases <a href="https://github.com/i3/i3/releases">here</a>.
```

Comment:
This is not a good substitute. Please use https://github.com/i3/i3/archive/master.tar.gz instead, which is the corresponding github link. The link gives you a snapshot of master, not just the released versions.